recipe/meta.yaml

```diff
@@ -142,8 +146,8 @@ requirements:
     - {{ pin_compatible('numpy') }}
```

Comment:
replace this with '>=1.21.2,<2' for py>=310 and otherwise for '>=1.19.2,<2'

Reply:
done